src/main.js

```diff
@@ -25,6 +25,7 @@ randomCoverButton.addEventListener('click', generateRandomCover);
 makeOwnCoverButton.addEventListener('click', viewMakeOwnForm);
 viewSavedButton.addEventListener('click', viewSavedCovers);
 homeButton.addEventListener('click', viewHomePage);
+makeMyBookButton.addEventListener('click', displayFormSubmissionCover)
```

Comment:
Great function name to clearly state what will happen when the user clicks this button.